3-async-await/solution.js

```diff
@@ -20,16 +20,43 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-function solution() {
+const productMethod = require('./products')
+const priceMethod = require('./prices')
+
+async function solution() {
     // YOUR SOLUTION GOES HERE
 
     // You generate your id value here
-
+    const id = parseInt(Date.now().toString().slice(-2))
```

Comment:
It could be a great idea to use a function that generate the ID's so if you need it later you can just invoke it. What do you think?
something like 
```
function idGenerator (){
}

```